### DISCUSSION TITLE
Exporting Suricata logs, PCAPS, and other info from SO

### DISCUSSION THREAD

**Discussion by Hamb0** [Security-Onion-Solutions/securityonion#7893] [Unsupported Versions] [answered]
I'm somewhat new to Security Onion, but we are trying to find a way to export Suricata logs, pcaps, and other additional data from SOM and into a csv or by other means for offloading and backing up for log retention up to 90 days. We are on an air-gapped and closed network, so any third party installations and access to internet is not feasible or possible. I am aware that you can do this is in the baked in version of Elastic with Kibana, and have done that before and know my way around it. But I do not believe that it contains the information that we need or want to keep for log retention. Any information that someone can provide would be greatly appreciated.

**Comment by dougburks** [comment, marked as answer]
One option might be to export the raw Suricata logs from `/nsm/suricata/`. If you're also using Zeek, you can find the raw Zeek logs in `/nsm/zeek/`.
